### DISCUSSION TITLE
How to connect with ruby part

### DISCUSSION THREAD

**Discussion by lc19997** [Junjie529/Golang-server-gin-#16] [Q&A] [answered]
I am not sure how I can connect with ruby signal hanlder.

**Comment by outdoorSpirit** [comment, marked as answer]
First, go to ruby signal generator.
export SERVER_ENDPOINT=http://localhost:31333/
export BEARER_TOKEN=EOtd08fod-EKxV8PU7jyNw
export DEMO_MODE=true

bundle exec ruby main.rb
